specs/ppd-search.spec

```diff
@@ -0,0 +1,222 @@
+* Deselect 'standard' from the 'Transaction category' checkbox list
+* Clicking show results will route User to the search results page
+* Summary will display '239 matching transactions'
+* First result item address displays as 'Rose Cottage, Gilson Road, Birmingham, B46 1LN'
+* Reset the form
+
+## Price range filter
+
+Tags: Search
+
+* Type "Rose Cottage" into the "Building name or number" field
+* Input '0' to the 'Minimum price' field
+* Input '200000' to the 'Maximum price' field
+* Clicking show results will route User to the search results page
+* Summary will display '4753 matching transactions'
+* First result item address displays as 'Rose Cottage, Malt Mill Lane, Halesowen, B62 8JE'
+* Reset the form
+
+## Date range filter
+
+Tags: Search
+
+* Type "Rose Cottage" into the "Building name or number" field
+* Select '22/11/2020' in the 'earliest' field
+* Select '22/11/2021' in the 'latest' field
+* Clicking show results will route User to the search results page
+* Summary will display '273 matching transactions'
+* First result item address displays as 'Rose Cottage, Malt Mill Lane, Halesowen, B62 8JE'
+* Reset the form
+
+## Small date range
+
+Tags: Search
+
+* Type "Rose Cottage" into the "Building name or number" field
+* Select '19/11/2021' in the 'earliest' field
+* Select '22/11/2021' in the 'latest' field
+* Clicking show results will route User to the search results page
+* Summary will display '0 transactions'
+* Reset the form
+
+## Invalid date range (earliest date occurs after latest date)
+
+Tags: Search
+
+* Type "Rose Cottage" into the "Building name or number" field
+* Select '22/11/2021' in the 'earliest' field
+* Select '22/11/2020' in the 'latest' field
+* Clicking show results will route User to the search results page
+* Summary will display '0 transactions'
+* Reset the form
+
+## Limit results to 1000
+
+Tags: Search
+
+* Type "Rose Cottage" into the "Building name or number" field
+* Select 'at most 1000' from the 'How many results?' radio list
+* Clicking show results will route User to the search results page
+* Summary will display '9873 matching transactions'
+* Summary will display '570 properties'
+* First result item address displays as 'Rose Cottage, Lower Gustard Wood, St Albans, AL4 8RU'
+* Reset the form
+
+## No Limit results
+
+Tags: Search
+
+* Type "Rose Cottage" into the "Building name or number" field
+* Select 'all' from the 'How many results?' radio list
+* Clicking show results will route User to the search results page
+* Summary will display '9873 matching transactions'
+* Summary will display '5887 properties'
+* A warning is visible with the phrase 'We have limited this page to 5000 results'
+* First result item address displays as 'Rose Cottage, Lower Gustard Wood, St Albans, AL4 8RU'
+* Reset the form
+
+## Large queries
+
+Tags: Search
+
+* Type "Birmingham" into the "Town or city" field
+* Select 'all' from the 'How many results?' radio list
```

Comment:
this is the kind of query where we'd like to have eyes-on the memory usage of the app as it's running